test/unit/test-runtime.js

```diff
@@ -415,9 +415,11 @@ describes.fakeWin(
     });
 
     it('should load correct extension version', async function () {
-      self.__AMP_MODE = {
-        rtvVersion: 'test-version',
-      };
+      // calculateExtensionScript isn't passed an opt_win arg
+      // so we can't use win.__AMP_MODE for both.
+      self.__AMP_MODE = {rtvVersion: 'test-version'};
+      win.__AMP_MODE = {test: false};
```

Comment:
FMI: Is it normal to have `__AMP_MODE` properties on multiple objects? Should we be able to expect them to be in sync?

Reply:
No! This is absolutely awful! The test is actually using two different Windows!